### DISCUSSION TITLE
Autofill should support firefox

### DISCUSSION THREAD

**Discussion by dyf** [AllenNeuralDynamics/aind-metadata-entry-js#122] [General]
**Describe the bug**
When pressing the "autofill" button on the metadata entry site on a firefox browser, it does not open local file system for schema selection.

**To Reproduce**
1. Open https://neural-dynamics-dev.uw.r.appspot.com/ on firefox
2. Click on autofill schema button
3. Click "OK" on prompt
5. See error 

**Expected behavior**
It should open local machine's file system to select a file

**Desktop (please complete the following information):**
 - Browser: Firefox

**Additional context**
User Report by JohnR

**Comment by jtyoung84** [comment]
See original note here: https://github.com/AllenNeuralDynamics/aind-metadata-entry-js/pull/37

the OpenFilePicker class being used isn't supported by FireFox

Here is an example using Firefox: 
https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications

**Comment by galenlynch** [comment]
Would be great to be able to use Firefox :)

**Comment by dyf** [comment]
@mekhlakapoor can you move this to a discussion thread and close this issue?

**Comment by mekhlakapoor** [comment]
Moved this to discussion thread
